spring-xd-module-spi/src/main/java/org/springframework/xd/module/options/mixins/FromMongoDbOptionMixin.java

```diff
@@ -40,7 +45,10 @@
 	private int pollRate = 1000;
 	
 	private int maxMessages = 1;
-	
+
+	private boolean split = true;
```

Comment:
We should add a comment to XD-2794 to give the contributor a heads-up that the source now splits by default.